### DISCUSSION TITLE
隐藏款啥时候上线

### DISCUSSION THREAD

**Discussion by bowencool** [ant-design/ant-design#35944] [Q&A] [answered]
这个 demo 在中文文档上找不到，只能通过 URL 锚点访问，是不是遗漏了？
https://ant.design/components/table-cn/#components-table-demo-resizable-column
![pk38SqFiDd](https://user-images.githubusercontent.com/20217146/172520992-257da070-0a19-4fde-861a-54bb30121531.png)

**Comment by bowencool** [comment]
顺便问下，有内置此功能的计划吗

**Reply by bowencool** [reply]
https://github.com/ant-design/ant-design/issues/33043#issuecomment-979677534

**Comment by MadCcc** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/27722486/172521528-c32a6df7-363f-49cc-a928-ecf9091d4e87.png)

**Comment by bowencool** [comment]
https://github.com/ant-design/ant-design/issues/32811
